ocs_ci/ocs/ui/views.py

```diff
@@ -1003,6 +1003,10 @@
     ),
     "failover-app": ("//button[normalize-space()='Failover application']", By.XPATH),
     "relocate-app": ("//button[normalize-space()='Relocate application']", By.XPATH),
+    "delete-app": ("//button[normalize-space()='Delete application']", By.XPATH),
+    "remove-app-resources": (".//*[@id='remove-app-resources']", By.XPATH),
+    "delete": ("//button[@type='button'][@class='pf-c-button pf-m-danger']", By.XPATH),
+    "no-results-found": ("//*[text()='No results found']", By.XPATH),
```

Comment:
Text `No results found` is already hardcoded here. What's the intention of using text() function then?

Reply:
can you please leave a suggestion here on how else this line could be modified ?

Reply:
Is the intention to always look for `No results found`? or is it a dynamic text?